config/core/ressources/mongodbsource.yaml

```diff
@@ -0,0 +1,139 @@
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    sources.google.com/release: devel
+    duck.knative.dev/source: "true"
+    knative.dev/crd-install: "true"
+  annotations:
+    registry.knative.dev/eventTypes: |
+      [
+        { "type": "mongodb.document.update" },
+        { "type": "mongodb.document.delete" }
+      ]
+  name: mongodb.sources.google.com
+spec:
+  group: sources.google.com
+  names:
+    categories:
+    - all
+    - knative
+    - eventing
+    - sources
+    kind: MongoDbSource
+    plural: mongodbsources
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        apiVersion:
+          type: string
+        kind:
+          type: string
+        metadata:
+          type: object
+        spec:
+          properties:
+            serviceAccountName:
+              type: string
+            secret:
+              type: localReferenceObject
```

Comment:
this type does not exist in openAPI

Reply:
You are right, I didnt know what to put there since it was local reference Object; maybe put the description of the localRefObj like tabbed and with collons?